src/Microsoft.ML.PCA/PcaTransform.cs

```diff
@@ -190,90 +281,104 @@ private static VersionInfo GetVersionInfo()
         {
             return new VersionInfo(
                 modelSignature: "PCA FUNC",
-                verWrittenCur: 0x00010001, // Initial
+                //verWrittenCur: 0x00010001, // Initial
+                verWrittenCur: 0x00010002, // Get rid of writing float size in model context
                 verReadableCur: 0x00010001,
```

Comment:
>0x00010001 [](start = 32, length = 10)

I think this also needs to change to 2

Reply:
@ivmatan@microsoft.com does verReadableCur need to be bumped up too? RffTransform didn't do it, so wanted to double check.

---
In reply to: [227069005](https://github.com/dotnet/machinelearning/pull/1333#discussion_r227069005) [](ancestors = 227069005)

Reply:
I think it's fine as it is. But let me double check.

---
In reply to: [227165477](https://github.com/dotnet/machinelearning/pull/1333#discussion_r227165477) [](ancestors = 227165477,227069005)

Reply:
Run it on top of old model, it loads fine. I think you good.

---
In reply to: [227485614](https://github.com/dotnet/machinelearning/pull/1333#discussion_r227485614) [](ancestors = 227485614,227165477,227069005)